lib/ESP/prusa/include/esp_tcp.h

```diff
@@ -49,6 +49,9 @@
 // It would be better to obtain this from ESP but there seems to be no way
 #define ESP_TCP_MSS 536
 
+// #define ALTCP_ESP_DEBUG      1
+#define ALTCP_ESP_EPCB_MAGIC 0xdeadf00d
```

Comment:
Should we enable debug by default now, or is there a downside to that (apart from little overhead)?

Reply:
Overhead is the only problem. I just didn't want the overhead to become part of the system. Maybe we can enable this as debugging is bound to quite heave prints. This limits chances it will stay enabled forever.